superset/security/manager.py

```diff
@@ -592,10 +592,8 @@ def clean_perms(self) -> None:
         sesh = self.get_session
         pvms = sesh.query(PermissionView).filter(
             or_(
-                PermissionView.permission  # pylint: disable=singleton-comparison
-                == None,
-                PermissionView.view_menu  # pylint: disable=singleton-comparison
-                == None,
+                PermissionView.permission is None,
+                PermissionView.view_menu is None,
```

Comment:
I don't think `is` triggers the SQLA comparator, therefore this might not generate the condition you want.

Reply:
![image](https://user-images.githubusercontent.com/335541/104784765-d8fccd80-573d-11eb-8554-4ec4829dfaaf.png)

Reply:
yeah I wanted to see Dialects supports this.